pkg/products/threescale/reconciler.go

```diff
@@ -705,10 +705,15 @@ func (r *Reconciler) reconcileExternalDatasources(ctx context.Context, serverCli
 	if err != nil {
 		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to create redis deletion failure alert for threescale: %w", err)
 	}
-	// create Redis CPU Usage High alert
+	// create Redis Memory Usage High alert
 	if err = resources.CreateRedisMemoryUsageAlerts(ctx, serverClient, r.installation, backendRedis); err != nil {
 		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to create backend redis prometheus memory usage high alerts for threescale: %s", err)
 	}
+
```

Comment:
@austincunningham we are doing a refactor which will move some of these places where checks are added to a function in metrics.go, shouldn't change what you're doing here too much but just an fyi. Will ping you on that pr when we have it open.

Reply:
Here's the pr https://github.com/integr8ly/integreatly-operator/pull/1108

Reply:
will hold off on this till #1108 is merged, not a blocker for us testing cpu utilization